torchsynth/module.py

```diff
@@ -152,34 +169,34 @@ def add_parameters(self, parameters: List[ModuleParameter]):
 
     def get_parameter(self, parameter_id: str) -> ModuleParameter:
         """
-        Get a single ModuleParameter for this module
+        Retrieves a single :class:`~torchsynth.parameter.ModuleParameter`, as
+        specified by its parameter Id.
 
         Args:
-            parameter_id: Id of the parameter to return
+            parameter_id: Id of the parameter to retrieve.
         """
         value = self.torchparameters[parameter_id]
         assert value.shape == (self.batch_size,)
         return value
 
     def get_parameter_0to1(self, parameter_id: str) -> T:
         """
-        Get the value of a parameter in the range of [0,1]
+        Retrieves a specified parameter value in the normalized range [0,1].
 
         Args:
-            parameter_id:   Id of the parameter to return the value for
+            parameter_id: Id of the parameter to retrieve.
         """
         value = self.torchparameters[parameter_id]
         assert value.shape == (self.batch_size,)
         return value
 
     def set_parameter(self, parameter_id: str, value: T):
         """
-        Update a specific parameter value, ensuring that it is within a specified
-        range
+        Updates a parameter value in a parameter-specific non-normalized range.
```

Comment:
We call the non-normalized range the human-readable range else where in the code base -- lets use that here as well? Or make a decision on the terminology (human-readable vs. machine-readable range) or (normalized vs. non-normalized range)

Reply:
I vote for 'human-readable vs. machine-readable range'